trainingMaquetado/Maquetado.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
```

Comment:
don't leave empty lines between tags.
Although it looks a bit uglier it's easier to recognize where an element ends.